bin/INSTALL_template.sh

```diff
@@ -357,8 +357,8 @@ find $HSS_INSTALL_DIR/hwgrading_website -type f -name \*.gif -exec chmod o+r {}
 # "other" can read & execute all .js files
 find $HSS_INSTALL_DIR/hwgrading_website -type f -name \*.js -exec chmod o+rx {} \;
 
-
-replace_fillin_variables $HSS_INSTALL_DIR/hwgrading_website/toolbox/configs/master.php
+replace_fillin_variables $HSS_INSTALL_DIR/hwgrading_website/toolbox/configs/master_template.php
+mv $HSS_INSTALL_DIR/hwgrading_website/toolbox/configs/master_template.php $HSS_INSTALL_DIR/hwgrading_website/toolbox/configs/master.php
```

Comment:
Extra step of moving a file after the replace variables?  Was this an intentional choice?

Hmm.  Maybe we should be doing this for all files that have variables replaced during configuration/installation?  It might help with the file timestamp bug that required I remove the "u" option for rsync.  Also, this might help prevent a potential glitch with running installation on a live server & seeing unreplaced variables?